java/ql/lib/semmle/code/java/Reflection.qll

```diff
@@ -380,11 +395,14 @@ class ReflectiveAnnotationAccess extends ClassMethodAccess {
   }
 }
 
+/** DEPRECATED: Alias for `ReflectiveAnnotationCall`. */
+deprecated class ReflectiveAnnotationAccess = ReflectiveAnnotationCall;
+
 /**
  * A call to `Class.getField(..)` that accesses a field.
  */
-class ReflectiveFieldAccess extends ClassMethodAccess {
-  ReflectiveFieldAccess() {
+class ReflectiveFieldCall extends ClassMethodCall {
```

Comment:
This class shouldn't be renamed. It's a call, yes, but it's a reflective access to a field, not a call to a field. The other subclasses of `ClassMethodCall` in this file should probably also not be renamed, as they're about accessing something via reflection and they're not calling whatever that is.